types/expo/expo-tests.tsx

```diff
@@ -484,6 +485,19 @@ async () => {
     isString(queryParams2['y'] || '');
 };
 
+// #region securestore
+async () => {
+    await SecureStore.setItemAsync('some-key', 'some-val', {
+        keychainAccessible: SecureStore.WHEN_PASSCODE_SET_THIS_DEVICE_ONLY,
+    });
+    const result = await SecureStore.getItemAsync('some-key');
+    if (result != null) {
+        result.slice() === 'some-val';
+    }
+    await SecureStore.deleteItemAsync('some-key');
+};
```

Comment:
Other `keychainAccessible` options can be tested here. Something like this:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/expo/expo-tests.tsx#L160